### ISSUE TITLE
[translation]: Update Portuguese translations for contributing documentations

### ISSUE THREAD

**Issue by raisedadead** [freeCodeCamp/freeCodeCamp#18309]
#### Update the following in Portuguese from their English version available in [`/docs`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs) directory:

- [ ] [`/docs/portuguese/CONTRIBUTING.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/portuguese/CONTRIBUTING.md) from its english version at [`/CONTRIBUTING.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/CONTRIBUTING.md)

- [ ] [`/docs/portuguese/README.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/portuguese/README.md) from its english version at [`/docs/README.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/README.md)

- [ ] [`/docs/portuguese/how-to-catch-outgoing-emails-locally.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/portuguese/how-to-catch-outgoing-emails-locally.md) from its english version at [`/docs/how-to-catch-outgoing-emails-locally.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/how-to-catch-outgoing-emails-locally.md)

- [ ] [`/docs/portuguese/how-to-setup-freecodecamp-locally.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/portuguese/how-to-setup-freecodecamp-locally.md) from its english version at [`/docs/how-to-setup-freecodecamp-locally.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/how-to-setup-freecodecamp-locally.md)

- [ ] [`/docs/portuguese/how-to-work-on-coding-challenges.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/portuguese/how-to-work-on-coding-challenges.md) from its english version at [`/docs/how-to-work-on-coding-challenges.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/how-to-work-on-coding-challenges.md)

- [ ] [`/docs/portuguese/how-to-work-on-guide-articles.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/portuguese/how-to-work-on-guide-articles.md) from its english version at [`/docs/how-to-work-on-guide-articles.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/how-to-work-on-guide-articles.md)
 
#### Once you are ready please make a pull request, preferably one for each file.

When you make a PR for this particular issue, do not tag it with `Closes #XXXXX` instead just reference it with `#XXXXX`

**Comment by raisedadead** [comment]
Awesome! All yours.

**Comment by raisedadead** [comment]
@JorgeAndd I would recommend starting one by one and making a separate PR for each. Also, please note that we rapidly iterating over the english versions at the moment.

So, take your time and revisit this issue thread for changes.

**Comment by JorgeAndd** [comment]
I will start working on **CONTRIBUTING.md**. If anyone else wants to help, feel free to.

**Comment by anacoimbrag** [comment]
Hey! I can translate the next file **how-to-catch-outgoing-emails-locally.md**

**Comment by thepurpleana** [comment]
Hey guys! I can translate the **how-to-work-on-coding-challenges.md** one :)

**Comment by lauraalpinto** [comment]
And I can finish up with **how-to-work-on-guide-articles.md**!  😊

**Comment by raisedadead** [comment]
There is still work pending on this. When you make a PR for this particular issue, do not tag it with `Closes #XXXXX` instead just reference it with `#XXXXX`

**Comment by ricardokojo** [comment]
I can work on **how-to-setup-freecodecamp-locally.md**.
Also, there is a substantial difference between PT-PT and PT-BR (Portuguese from Portugal and Portuguese from Brazil). Maybe there should be different translations for each one?

**Comment by arbolitoloco** [comment]
I just noticed there are versions of things in both Portuguese and Brazilian Portuguese. Hope that's OK.

**Comment by myakiju** [comment]
> I can work on **how-to-setup-freecodecamp-locally.md**.
> Also, there is a substantial difference between PT-PT and PT-BR (Portuguese from Portugal and Portuguese from Brazil). Maybe there should be different translations for each one?

I agree with @ricardokojo. PT-PT and PT-BR have many differences and it would be nice to have both made separately

**Comment by lauraalpinto** [comment]
There are some differences but it is very easy to understand for both PT-BR and PT-PT speakers, in my opinion. I don't mind reading in PT-BR and I would guess the opposite also happens?

**Comment by raisedadead** [comment]
> PT-PT and PT-BR have many differences and it would be nice to have both made separately

Hi All, 

Thanks a lot for taking the time to contribute to freeCodeCamp.org

At the minute we are making the translations available in the PT-BR. We consider PT-PT in future, but I am afraid we don.t have an ETA on that.

Thanks.

**Comment by marceloFA** [comment]
Is there still missing translations for this issue?

**Comment by danilogila** [comment]
Is there any missing translation? I would like to help

**Comment by theogoulart** [comment]
I'll be helping on this one. 

Just one question, should we translate PT-PT to PT-BR? For example, in PT-PT "corre" means _execute_, but it has totally different meaning in PT-BR (_sprint_). In this case, I would substitute it by "executa", the correct translation.

**Comment by stearruda** [comment]
Is the issue completed? I can also help with any Portuguese translations.

**Comment by sofiabsilva** [comment]
I can help as well, if it's still needed.

**Comment by terumiisobe** [comment]
Hey! I'm gonna start working on **how-to-work-on-guide-articles.md** !

**Comment by demetrius99** [comment]
Hello folks, I can assist with PT-BR

**Comment by Belz** [comment]
Hi!
I fixed links on the portuguese README.md and took the chance to also do it on the english version (links did not work).

**Comment by stearruda** [comment]
There are 2 docs not listed in the Issue, but with no PT-BR translation. 

- [style-guide-for-curriculum-challenges.md](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/style-guide-for-curriculum-challenges.md)
- [style-guide-for-guide-articles.md](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/style-guide-for-guide-articles.md)

Can I get one of those?

**Comment by flauberjp** [comment]
hey @stearruda, what about us divide this one. I could pick the second one: [style-guide-for-guide-articles.md](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/style-guide-for-guide-articles.md).

**Comment by raisedadead** [comment]
Thanks for working on the translations for the "Contributing to freeCodeCamp". Here on we are making contributing to translations for contribution docs *ad hoc*. 

You are free to make pull-requests for improving these translations. We are closing the issue as these do not need tracking per se.

You can raise a fresh issue if you find something incorrect in the existing docs.
